src/ui/backend/BaseBacking.js

```diff
@@ -62,36 +83,39 @@ export default class BaseBacking {
 
   copy () {
     var copy = {};
+    for (var key in BASE_STYLE_PROPS) {
```

Comment:
Historically, I know `for (var ... in ...)` was slower than iterating over a set length array, have modern JIT made this negligible or is it will worth breaking these out into an array of keys like `styleKeyList` was doing?